form.js

```diff
@@ -0,0 +1,68 @@
+const form = document.querySelector('[data-js="form"]');
+const main = document.querySelector('[data-js="main"]');
+const inputQuestion = document.querySelector('[data-js="question"]');
+const inputAnswer = document.querySelector('[data-js="answer"]');
+const inputTag1 = document.querySelector('[data-js="tag1"]');
+const inputTag2 = document.querySelector('[data-js="tag2"]');
+const inputTag3 = document.querySelector('[data-js="tag3"]');
+
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
+
+  const formData = new FormData(event.target);
+  const dataObject = Object.fromEntries(formData);
+  console.log(dataObject);
```

Comment:
A tiny little `console.log` is here. ;-P 
Highly recommend removing it in production code.